metricbeat/module/elasticsearch/ccr/_meta/docs.asciidoc

```diff
@@ -1,3 +1,8 @@
 This is the `ccr` metricset of the Elasticsearch module. It interrogates the
-Cross Cluster Replication Stats API endpoint to fetch information about shards
-in the Elasticsearch cluster that are participating in cross-cluster replication.
+Cross-Cluster Replication Stats API endpoint to fetch metrics about cross-cluster
+replication from the Elasticsearch clusters that are participating in cross-cluster
+replication.
+
+If the Elasticserach cluster does not have the cross-cluster replication feature
```

Comment:
I'd suggest pruning a few words here and saying: "...does not have cross-cluster replication enabled..."